src/components/common/index.css

```diff
@@ -0,0 +1,70 @@
+.navBar {
+  width: 100%;
+  height: 60px;
+
+  background: linear-gradient(
+    90deg,
+    rgba(0, 72, 145, 1) 0%,
+    rgba(38, 120, 204, 1) 100%
+  ) !important;
+  background: -moz-linear-gradient(
```

Comment:
generally in modern days we don't need this `-moz` and `-webkit` stuff, so you can delete linse 10-19